src/db/schema.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  sqliteTable,
+  integer,
+  text,
+  foreignKey
+} from 'drizzle-orm/sqlite-core';
+
+// ---------------------------------------
+// TEACHER
+// ---------------------------------------
+export const teacher = sqliteTable('teacher', {
+  id: integer('id', { mode: 'number' }).primaryKey({ autoIncrement: true }),
+  first_name: text('first_name').notNull(),  // First name
+  last_name: text('last_name').notNull(),    // Last name
+  email: text('email').notNull(),            // Email
+  phone_number: text('phone_number'),        // Phone number
+  avatar: text('avatar'),                    // Avatar
+  priority: integer('priority')              // Priority
```

Comment:
Macht es Sinn, die Priorität als ein float zu machen? Hier ein Use-Case: es gibt 10 Lehrer mit der Priorität 1. Nun hat der eine Lehrer doch mehr Priorität bekommen - dann muss man bei 9 Lehrern die Priorität zu 2 ändern. Bei einem Float könnte man dann einfach es auf 0.9 wechseln. Wie sieht ihr es?

Reply:
Ich denke nicht das wir es als Float machen sollten. Ich finde man muss sich irgendwie einschränken wie die Priorisierung ist sonst wird das irgendwie unschön weil dann ganz schnell immer kleinerer Abstände genommen werden.

Alternativ könnten wir die Priorisierung auch irgendwie dynamisch machen, in dem man die Lehrer einfach anordnen bzw auf eine Ebene setzen kann und daraus im Hintergrund die Priorisierung berechnet wird.